### PR TITLE
Implement proper error handling

### LINKED ISSUE #55: Error handling
If an exception is raised during validation (#50) or execution of a query, that error needs to be caught and returned to the user in the response.

## Requirements

For basic requirements, see the [GraphQL spec section 7.2.2](http://facebook.github.io/graphql/October2016/#sec-Errors). Additional requirements for this project:

1. Handle arbitrary python exceptions.
2. Optionally, define and custom errors that can be raised during validation or execution, if that would be useful for providing additional information.

## Exclusions

- We do not need to associate any errors with particular points unless that information is almost harder to not include :-)

### PR DESCRIPTION
**Pull request by ariannedee**
Fixes #55

### CODE REVIEW

File: django_graph_api/tests/test_errors.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+def test_multiple_queries(starwars_data):
+    document = '''
+        mutation MyMutation {
+          episodes {
+            name
+          }
+        }
+        '''
```
Comment (melinath) [inline django_graph_api/tests/test_errors.py]: This fwiw is testing what happens when zero queries are defined (which is currently same behavior as multiple queries).

**Comment by melinath** [comment]
This also fixes #63.

File: django_graph_api/tests/test_errors.py (hunk 1 above)
Comment (ariannedee) [reply django_graph_api/tests/test_errors.py]: Oh yeah, I forgot to change the name 👍

**Comment by ariannedee** [comment]
> In principal this looks good to me, but I can't run the tests because graphql.error doesn't exist. Travis is having issues as well. It seems like this might be a relative vs. absolute import issue?

Actual problem was much bigger/stranger. Apparently the graphql library that my IDE was looking at was graphql-core. I had remembered we used a simple parsing/lexing library, but then thought that maybe someone had swapped it out before and I didn't notice. So I was using exceptions from the wrong library.

I used subclassed the graphql-core one instead of just defining a simple one myself because I thought that their location stuff m be useful in the future. I think it's better this way anyways...

Also Travis is having long delays right now, but it should pass. I would like a slack channel for build statuses and will look into it. Or an email if one of my builds fail, since it is starting to take a little while now.

**Comment by ariannedee** [comment]
@melinath Tests passing and I updated the test name.

**Pull request merged**